vignettes/modules.Rmd

```diff
@@ -30,19 +30,20 @@ library(mhealthtools)
 library(dplyr)
 ```
 
-#### Walk Module
+### Walk Module
 
 In this task, the participant is asked to keep the phone in his/her pant's front pocket (if not in the waistband of the pants) and then walk in a straight line for some fixed amount of time (like 30s). For someone with Parkinson's we expect to see symptoms of Dyskinesia and Bradykinesia. 
 
-The features extracted from `get_walk_features` do capture to some extent the effects of those symptoms. In the default processing, the signal is broken down into IMFs (Intrinsic Mode Functions) using the Hilbert Huang Transform before they are analyzed.
+The features extracted from `get_walk_features` do capture to some extent the effects of those symptoms. In the default processing, the signal is broken down into two IMFs (Intrinsic Mode Functions) using the Hilbert Huang Transform before they are analyzed.
 
-`mhealthtools` comes with preloaded walk data from one of the mpower participants. Let's take a look at the data before we proceed to analyze it.
+`mhealthtools` comes with preloaded sample walk data from the mpower study. Let's take a look at the walk data before we proceed to analyze it.
 
 ```{r}
```

Comment:
Please make the changes I referenced in #144 and I will do a full review of the modules vignette from there.

Reply:
(This is in reference to the call to `data(...)` below as well as other changes that still need to be made in this document).